oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/pom.xml

```diff
@@ -48,6 +48,11 @@
 <!--        <artifactId>mysql-connector-java</artifactId>-->
 <!--        <version>8.0.13</version>-->
 <!--        </dependency>-->
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.2.18</version>
```

Comment:
version has to define in the properties block.

Reply:
Version and dependency should be managed in the oap root pom.
Dependency Management of pom.xml